pkgs/unified_analytics/test/unified_analytics_test.dart

```diff
@@ -122,6 +123,7 @@ void main() {
       hostOsVersion: hostOsVersion,
       locale: locale,
       clientIde: clientIde,
+      enabledFeatures: enabledFeatures,
```

Comment:
Does this mean every event across every tool will send this "enabledFeatures" field?

Reply:
Ahh ok, i see the comment [here](https://github.com/dart-lang/tools/issues/198#issuecomment-1840821591)